docs/SETUP_REMOTE.md

```diff
@@ -29,6 +29,14 @@ There are other options like a Docker container per person though.
 
 Install tools you need to perform remote processing.
 
+If some tool require shell initialization scripts or special environment variables you should add them to your `~/.bashrc` or `~/.bash_profile` file depending on your remote operating system.
+
+Be sure to add all your configurations at the beginning of the file before this line :
+
+  ```bash
+  # If not running interactively, don't do anything
```

Comment:
Basically you narrow the texting to Ubuntu users only if I understand correctly. No one else ships default configs, I don’t recall such practice.